Example of usage with Tcl (#1349)
yuzutech/kroki: docs/modules/setup/pages/encode-diagram.adoc

- xref:kotlin[]
- xref:go[]
- xref:php[]

> - xref:Tcl[]

Reviewer: ```suggestion
- xref:tcl[]
```

----
include::example$encode.php[]
----

> [#Tcl]

Reviewer: ```suggestion
[#tcl]
```

== Tcl

> [source,Tcl]

Reviewer: ```suggestion
[source,tcl]
```

----
# assuming the diagram code is stored in the variable named `diagramSource`:
puts [string map {+ - / _} [binary encode base64 [zlib compress $diagramSource]]]
----

> Also see the Tcl package {uri-dia2kroki}[dia2kroki] which has proceudres to both decode and encode the diagram source code and a GUI to create diagrams and see the result directly. 

Reviewer: ```suggestion
Also, see the Tcl package {uri-dia2kroki}[dia2kroki] which has procedures to both decode and encode the diagram source code and a GUI to create diagrams with a live preview. 
```
